Xamarin.Forms.DataGrid/DataGrid.xaml.cs

```diff
@@ -315,12 +294,6 @@ public Thickness BorderThickness
 			set { SetValue(BorderThicknessProperty, value); }
 		}
 
```

Comment:
You shouldn't remove any property in existing API. That kind of action would break end user's code.